src/components/modal/_macro.njk

```diff
@@ -1,19 +1,19 @@
 {% macro onsModal(params) %}
-    {% set modalID = params.id | default('dialog') %}
-    <dialog class="ons-modal ons-js-modal {{ params.classes }}"
-        id="{{ modalID }}"
+    {% set modalId = params.id | default('dialog') %}
+    <dialog class="ons-modal ons-js-modal {{ params.classes if params.classes is defined and params.classes else '' }}"
```

Comment:
If the space before the `{{` here was added inside it wouldn't add a space to the class list